<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEdit</class>
 <widget class="QDialog" name="DialogEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Property</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutDialogEdit">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDialogEditContent">
     <item>
      <layout class="QGridLayout" name="gridLayoutDialogEdit1">
       <item row="1" column="0">
        <widget class="QLabel" name="labelPropertyDocName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Doc Name:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditPropertyDocName</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEditPropertyDefaultValue">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelPropertyDocDetail">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Doc Detail:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="buddy">
          <cstring>lineEditPropertyDocDetail</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditPropertyDocDetail">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelPropertyType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Type:</string>
         </property>
         <property name="buddy">
          <cstring>comboBoxPropertyType</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditPropertyName">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelPropertyDefaultValue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Default Value:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditPropertyDefaultValue</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelPropertyName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditPropertyName</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelPropertyTypeStringName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Type String Name:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditPropertyTypeStringName</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="spinBoxPropertyRevision"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditPropertyDocName">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelPropertyRevision">
         <property name="text">
          <string>Revision:</string>
         </property>
         <property name="buddy">
          <cstring>spinBoxPropertyRevision</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEditPropertyTypeStringName">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBoxPropertyType"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelPropertyDocBrief">
         <property name="text">
          <string>Doc Brief:</string>
         </property>
         <property name="buddy">
          <cstring>lineEditPropertyDocBrief</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditPropertyDocBrief"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayoutDialogEdit2">
       <item row="6" column="1">
        <widget class="QCheckBox" name="checkBoxPropertyDesignable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPropertyRead">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Read:</string>
         </property>
         <property name="buddy">
          <cstring>checkBoxPropertyRead</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelPropertyReset">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Reset:</string>
         </property>
         <property name="buddy">
          <cstring>checkBoxPropertyReset</cstring>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelPropertyEnabled">
         <property name="text">
          <string>Enabled:</string>
         </property>
         <property name="buddy">
          <cstring>checkBoxPropertyEnabled</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="checkBoxPropertyEnabled">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="labelPropertyUser">
         <property name="text">
          <string>User:</string>
         </property>
         <property name="buddy">
          <cstring>checkBoxPropertyUser</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelPropertyWrite">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Write:</string>
         </property>
         <property name="buddy">
          <cstring>checkBoxPropertyWrite</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="checkBoxPropertyNotify">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="checkBoxPropertyStored">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelPropertyNotify">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Notify:</string>
         </property>
         <property name="buddy">
          <cstring>checkBoxPropertyNotify</cstring>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QCheckBox" name="checkBoxPropertyUser">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QCheckBox" name="checkBoxPropertyConstant">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelPropertyScriptable">
         <property name="text">
          <string>Scriptable:</string>
         </property>
         <property name="buddy">
          <cstring>checkBoxPropertyScriptable</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="checkBoxPropertyWrite">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="labelPropertyStored">
         <property name="text">
          <string>Stored:</string>
         </property>
         <property name="buddy">
          <cstring>checkBoxPropertyStored</cstring>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="labelPropertyConstant">
         <property name="text">
          <string>Constant:</string>
         </property>
         <property name="buddy">
          <cstring>checkBoxPropertyConstant</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelPropertyMember">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Member:</string>
         </property>
         <property name="buddy">
          <cstring>checkBoxPropertyMember</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBoxPropertyMember">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="checkBoxPropertyReset">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBoxPropertyRead">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="checkBoxPropertyScriptable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelPropertyDesignable">
         <property name="text">
          <string>Designable:</string>
         </property>
         <property name="buddy">
          <cstring>checkBoxPropertyDesignable</cstring>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="labelPropertyFinal">
         <property name="text">
          <string>Final:</string>
         </property>
         <property name="buddy">
          <cstring>checkBoxPropertyFinal</cstring>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QCheckBox" name="checkBoxPropertyFinal">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDialogEditButtons">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSaveProperty">
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCloseProperty">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditPropertyName</tabstop>
  <tabstop>lineEditPropertyDocName</tabstop>
  <tabstop>lineEditPropertyDocBrief</tabstop>
  <tabstop>lineEditPropertyDocDetail</tabstop>
  <tabstop>comboBoxPropertyType</tabstop>
  <tabstop>lineEditPropertyTypeStringName</tabstop>
  <tabstop>lineEditPropertyDefaultValue</tabstop>
  <tabstop>spinBoxPropertyRevision</tabstop>
  <tabstop>checkBoxPropertyMember</tabstop>
  <tabstop>checkBoxPropertyRead</tabstop>
  <tabstop>checkBoxPropertyWrite</tabstop>
  <tabstop>checkBoxPropertyReset</tabstop>
  <tabstop>checkBoxPropertyNotify</tabstop>
  <tabstop>checkBoxPropertyEnabled</tabstop>
  <tabstop>checkBoxPropertyDesignable</tabstop>
  <tabstop>checkBoxPropertyScriptable</tabstop>
  <tabstop>checkBoxPropertyStored</tabstop>
  <tabstop>checkBoxPropertyUser</tabstop>
  <tabstop>checkBoxPropertyConstant</tabstop>
  <tabstop>checkBoxPropertyFinal</tabstop>
  <tabstop>pushButtonSaveProperty</tabstop>
  <tabstop>pushButtonCloseProperty</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
